src/applications/static-pages/BTSSS-login/selectors.js

```diff
@@ -1,14 +1,8 @@
-import { createSelector } from 'reselect';
 import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
 import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
 
-const selectFeatureToggles = createSelector(
-  state => ({
-    BTSSSLoginWidget: toggleValues(state)[FEATURE_FLAG_NAMES.BTSSSLoginWidget],
-  }),
-  toggles => toggles,
-);
+const selectFeatureToggles = state => {
+  return toggleValues(state)[FEATURE_FLAG_NAMES.btsssLoginWidget];
+};
```

Comment:
This is more concise and appropriate, in this case, than the use of `createSelector` 👍 

**CONSIDER**
It might feel overly critical, but it seems to me like the plural name `selectFeatureToggles` is a bit inaccurate and confusing. I think a singular name would be more appropriate and clear (`selectFeatureToggle`).

Reply:
I agree. Fixed.